CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+# 92.2.0
+
+This minor release adds new features to MDCAlertController and MDCTabBarView.
+
+## New features
+
+MDCAlertController has a new accessoryView property. This property can be used to add custom views
+to the alert controller. Example usage:
+
+```swift
+let alert = MDCAlertController(title: "This is a title", message: "This is a message")
+let textField = UITextField()
+textField.placeholder = "This is a text field"
+alert.accessoryView = textField
+``` 
+
+TabBarView has a new `setContentPadding:forLayoutStyle:` API. Example usage:
+
+```objc
+tabBarView.preferredLayoutStyle = MDCTabBarViewLayoutStyleFixed;
```

Comment:
Tip: Do you want to demonstrate that it's not for the `preferredLayoutStyle`? The most common use case is actually to adjust the leading inset for `.Scrollable`, since the default 52 points is significant on smaller form factors.

Reply:
Changed the example to use MDCTabBarViewLayoutStyleScrollable.